index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <link rel="stylesheet" href="css/style.css">
+  <body>
+    <nav id="navbar"><h1>  Firts Navbar</h1>
```

Comment:
Header should come at the header tag, instead

Reply:
@for-ashraf It would be better if you explain more  why it should be using the header tag, it may be obvious for you but not for everyone so please try to do that

Reply:
The use of Header tags makes it more organized. Semantic HTML refers to syntax that makes the HTML more comprehensible by better defining the different sections and layout of web pages. It makes web pages more informative and adaptable,